src/goto-instrument/goto_instrument_parse_options.cpp

```diff
@@ -1779,6 +1800,8 @@ void goto_instrument_parse_optionst::help()
     " --drop-unused-functions      drop functions trivially unreachable from main function\n" // NOLINT(*)
     " --print-internal-representation\n" // NOLINTNEXTLINE(*)
     "                              show verbose internal representation of the program\n"
+    " --print-detailed-ir\n" // NOLINTNEXTLINE(*)
```

Comment:
Are there tests for this?

Reply:
We were using this for our internal tests. It shouldn't be in the PR, removed it.